src/js/claims-status/components/appeals-v2/Expander.jsx

```diff
@@ -13,13 +23,15 @@ const Expander = ({ title, dateRange, onToggle, cssClass, hideSeparator }) => {
         <h3 style={{ color: 'inherit' }}>{title}</h3>
       </button>
       <div className="appeal-event-date">{dateRange}</div>
+      {expanded && alert}
```

Comment:
Took me a bit to realize that `alert` doesn't _always_ show when the timeline is expanded, because it's either content or `null`. Might be worth doing something like 
`const alert = (missingEvents && expanded) ? missingEventsAlert : null;`
`return ( <stuff...>{alert}</stuff>)`
to keep all of the logic in the same place. Or not, I can't really decide 🤷‍♂️

Reply:
I'll go through and see if I can clarify it.